test/test-remote/test_loki_log_ingest_api.ts

```diff
@@ -352,8 +352,7 @@ suite("Loki API test suite", function () {
     const testname = testName(this);
 
     // Specify details of log record to be inserted.
-    const sampleTimeRFC3339nano = "2012-10-10T10:10:01.123456789Z";
-    const sampletimestamp = ZonedDateTime.parse(sampleTimeRFC3339nano);
+    const sampletimestamp = ZonedDateTime.now().minusMinutes(10);
```

Comment:
update: see discussion above. This change makes sense because we do not want to test inserting data older than the retention period.